react/ProductList.tsx

```diff
@@ -28,15 +28,34 @@ const ProductList: FC<Props> = ({ renderAsChildren, classes }) => {
 
   const handleQuantityChange = useCallback(
     (uniqueId: string, quantity: number, item: OrderFormItem) => {
-      const adjustedItem = {
-        ...mapCartItemToPixel(item),
-        quantity,
+      if (quantity === item.quantity) {
+        return
+      }
+
+      const quantityIncreased = quantity > item.quantity
+
+      if (quantityIncreased) {
+        const adjustedItem = {
+          ...mapCartItemToPixel(item),
+          quantity: quantity - item.quantity,
```

Comment:
Shouldn't this be the newest product quantity?

Reply:
I thought so too, but no. AJ said this is the right way. Also, doing this way makes it consistent with the `addToCart` event sent by the AddToCartButton, that sends not the final product quantity, but the amount that was added.

Reply:
Oookay!